assets/js/vue-apps/past-grants.js

```diff
@@ -45,14 +45,47 @@ function init() {
             // @ts-ignore
             const initialData = window._PAST_GRANTS_SEARCH;
             const initialQueryParams = get(initialData, 'queryParams', {}) || {};
+            const facets = get(initialData, 'facets', {});
+
+            // Reconstruct the summary by grabbing labels from facets
+            // (eg. for the subset of filters which have different labels
+            // to their URL-provided values
+            const initialFilterSummary = () => {
```

Comment:
This if in this function is always true right? If so you might be able use `lodash/map` over the initialQuery params stuff rather than pushing to a new array. Could possibly be something like this:

```suggestion
const initialFilterSummary = map(initialQueryParams, (value, key) => {
    let label;
    switch (key) {
        case 'amount':
            label = get(facets, 'amountAwarded[0].label');
            break;
        case 'awardDate':
            label = get(facets, 'awardDate[0].label');
            break;
        case 'localAuthority':
            label = get(facets, 'localAuthorities[0].label');
            break;
        case 'westminsterConstituency':
            label = get(facets, 'westminsterConstituencies[0].label');
            break;
    }

    return {
        name: key,
        label: label || value
    };
});
```

Reply:
Good call!